cmd/broker/main.go

```diff
@@ -81,10 +89,20 @@ func runWithContext(ctx context.Context) error {
 
 	ctrlr := controller.CreateController(namespace, k8sClient, osClient)
 
-	ctrlr.RegisterDeployer(fuse.NewDeployer("fuse-deployer"))
-	ctrlr.RegisterDeployer(launcher.NewDeployer("launcher-deployer"))
-	ctrlr.RegisterDeployer(che.NewDeployer("che-deployer"))
-	ctrlr.RegisterDeployer(threescale.NewDeployer("3scale-deployer"))
+
+
+	if shouldRegisterService( fuseOnlineServiceName) {
```

Comment:
Is it possible to apply some DRY here?

Reply:
The only way I would think to do that is to add an array with the service names and a for loop. Is that what you're thinking. Not sure we gain a lot from that

Reply:
Ahh I think you meant in the switch statement where I forgot to update to use the constant